diff --color --unified --recursive --text jasp-desktop-0.16.2/Tools/CMake/R.cmake jasp-desktop-0.16.21/Tools/CMake/R.cmake
--- jasp-desktop-0.16.2/Tools/CMake/R.cmake	2022-05-29 16:06:19.219177777 +0800
+++ jasp-desktop-0.16.21/Tools/CMake/R.cmake	2022-05-29 15:56:46.254247812 +0800
@@ -751,7 +751,9 @@
   set(RINSIDE_PATH "${R_LIBRARY_PATH}/RInside")
   set(RENV_PATH "${R_LIBRARY_PATH}/renv")
 
+  if(NOT USE_LOCAL_R_LIBS_PATH)
   set(USE_LOCAL_R_LIBS_PATH ", lib='${R_LIBRARY_PATH}'")
+  endif(NOT USE_LOCAL_R_LIBS_PATH)
 
   message(CHECK_START "Looking for R.h")
   set(R_INCLUDE_PATH "${R_HOME_PATH}/include")
